src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -714,21 +662,23 @@ OverlayImpl::onManifests (
     {
         auto& s = m->list ().Get (i).stobject ();
 
-        if (auto mo = make_Manifest (s))
+        if (auto mo = Manifest::make_Manifest (s))
         {
             uint256 const hash = mo->hash ();
             if (!hashRouter.addSuppressionPeer (hash, from->id ()))
                 continue;
 
             auto const serialized = mo->serialized;
-            auto const result = manifestCache_.applyManifest (
+            auto const result = app_.manifestCache ().applyManifest (
                 std::move(*mo),
-                app_.validators(),
-                journal);
+                app_.validators());
 
             if (result == ManifestDisposition::accepted ||
                     result == ManifestDisposition::untrusted)
-                app_.getOPs().pubManifest (*make_Manifest(serialized));
+            {
+                app_.getOPs().pubManifest (
+                    *Manifest::make_Manifest(serialized));
```

Comment:
I think the `Manifest` isn't passed in to `applyManifest` as a const reference so that it can be stored on the cache via `std::move`.
https://github.com/wilsonianb/rippled/blob/dynamic-unl/src/ripple/app/misc/impl/Manifest.cpp#L250
We've been passing `Manifest`s to `applyManifest` with `std::move`, but there are some cases (like here) that it'd be nice to still have the `Manifest` around.